### PR TITLE
Client creation in NewFramework

### PR DESCRIPTION
**Pull request by awels**
- Moved client creation into NewFramework instead of BeforeEach
  block.

Signed-off-by: Alexander Wels <awels@redhat.com>

### CODE REVIEW

**Comment by awels** [comment]
ci please test

**Comment by awels** [comment]
ci test please

File: tests/framework/framework.go [hunk 1]
```diff
@@ -129,27 +129,28 @@ func NewFramework(prefix string, config Config) (*Framework, error) {
...
 
+	// clients
+	kcs, err := f.GetKubeClient()
+	if err != nil {
+		err = errors.Wrap(err, "ERROR, unable to create K8SClient")
+	} else {
+		f.K8sClient = kcs
+	}
+
+	cs, err := f.GetCdiClient()
```
Comment (jeffvance) [inline tests/framework/framework.go]: If you fail to create the kubeClient above, do you still want to create the cdi client here? I expected to see a pattern like this:
```
	kcs, err := f.GetKubeClient()
 	if err != nil {
 		return nil, errors.Wrap(err, "ERROR, unable to create K8SClient")
 	f.K8sClient = kcs
...
```

File: tests/framework/framework.go [hunk 2]
```diff
@@ -129,27 +129,28 @@ func NewFramework(prefix string, config Config) (*Framework, error) {
...
+	if err != nil {
```
Comment (jeffvance) [inline tests/framework/framework.go]: ditto

File: tests/framework/framework.go (hunk 1 above)
Comment (awels) [reply tests/framework/framework.go]: You are right, I will push follow up soon.

**Pull request merged**